### PR TITLE
Replace  all password type fields to certificate type for consistency

### PR DESCRIPTION
**Pull request by mdazam1942**

### CODE REVIEW

File: stix_shifter_modules/alertflex/configuration/config.json [hunk 1]
```diff
@@ -22,18 +22,18 @@
...
-            "type": "password",
+            "type": "certificate",
             "optional": true
         }
     },
     "configuration": {
         "auth": {
             "type" : "fields",
             "username": {
-                "type": "password"
```
Comment (delliott90) [inline stix_shifter_modules/alertflex/configuration/config.json:33]: This should not even be hidden as a password but be plain text I would think.

**Comment by delliott90** [comment]
We don't want all passwords changed over, just the actual certificate fields.

**Comment by mdazam1942** [comment]
> We don't want all passwords changed over, just the actual certificate fields.

ohk the issue description mislead me. will have to revert back all these change.

**Comment by mdazam1942** [comment]
closing this pr. reverting back is more work than just creating new PR  #1000

**Pull request closed without merging**